index.html

```diff
@@ -225,5 +225,27 @@ <h3>Skills</h3>
             </ul>
             </div>
         </section>
+        <section class="contact">
+            <div>
+            <h2>I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.
+            </h2>
+            <form action="https://formspree.io/f/{https://formspree.io/f/xeqdywdz}" method="post">
```

Comment:
- [ ] The closing tag in the action attribute should be removed as it would confuse the link.

Reply:
Good job making changes as required however, your form is not submitting.  On clicking submit button it should reroute to the formspree success page below. 
<img width="1669" alt="Screenshot 2022-12-13 at 13 23 21" src="https://user-images.githubusercontent.com/48546992/207292566-fd2c043e-d17b-4edc-ba9c-cf82fbcab3e7.png">
This has to be with how you are using the service. I suggest you remove the last curly brace on the action
 `<form
          action="https://formspree.io/f/xeqdywdz"
          method="POST"
        >`
The above way is how you use formspree. You have a redundant `https call `